### PR TITLE
Integrate pangolin lineages into virologist report

### PR DESCRIPTION
**Pull request by fxwiegand**
This PR will integrate the pangolin lineages from [here](https://github.com/cov-lineages/pangolin/tree/master/pangolin/data) into the virologist report by checking which alterations from pangolin occur for each sample.

### CODE REVIEW

**Comment by johanneskoester** [comment]
Next to each checkmark, can you please add the allele frequency (field AF) in parentheses?

File: workflow/scripts/generate_virologist_output.py [hunk 1]
```diff
@@ -191,6 +211,34 @@
...
+table["z"][1].append("ORF1ab:N1094_G1095delinsR:0.246") # Not contained in pangolin lineages
+
+"""Iterate variants of interest and other variants and check if gene and alteration are contained in pangolin data."""
+for sample in table:
+    lineage = table[sample][0][0].lower()
+    print(f"Checking lineage: {lineage}")
+    if lineage in pangolin_lineages:
+        for pangolin_variant in pangolin_lineages[lineage]:
+            pan_gene = pangolin_variant.split(":")[1]
+            pan_alteration = pangolin_variant.split(":")[2]
```
Comment (johanneskoester) [inline workflow/scripts/generate_virologist_output.py:227]:
```suggestion
            pan_gene, pan_alteration = pangolin_variant.split(":")
```

File: workflow/scripts/generate_virologist_output.py [hunk 2]
```diff
@@ -191,6 +211,34 @@
...
+            entry = f"{pan_gene}:{pan_alteration}:false"
+            for variant in itertools.chain(table[sample][1], table[sample][2]):
+                gene = variant.split(":")[0]
+                alteration = variant.split(":")[1]
+                vaf = variant.split(":")[2]
```
Comment (johanneskoester) [inline workflow/scripts/generate_virologist_output.py:232]:
```suggestion
                gene, alteration, vaf = variant.split(":")
```

**Review by johanneskoester — CHANGES_REQUESTED** [review]

File: resources/report-table-formatter.js [hunk 3]
```diff
@@ -1,13 +1,64 @@
...
     })
 
     if (value !== "no strain called") {
-        var lineage = value.split(' ')[0];
+        var split = value.split(' ');
+        var lineage = split[0];
         var link = `<a data-toggle="tooltip" data-placement="top" title="Linkout to cov-lineages" href='https://cov-lineages.org/lineages/lineage_${lineage}.html' target='_blank'>${lineage}</a>`;
-        return link + value.split(lineage).pop();
+
+        const table = `<div style="height: 200px; overflow-y: auto; white-space:pre-wrap;"><table class="table">
```
Comment (johanneskoester) [inline resources/report-table-formatter.js:16]: Not for this PR, but in a follow-up, I'd like you to replace these implementations here with a [nunjucks](http://mozilla.github.io/nunjucks) based templating.

Comment (fxwiegand) [reply resources/report-table-formatter.js:16]: That looks pretty cool and really makes sense to me. All that juggling around with string and then concatenating them all back together feels just "wrong".

**Review by johanneskoester — APPROVED** [review]

**Pull request merged**